README.md

```diff
@@ -173,46 +99,13 @@ logging config format for content details.
 **Docker (Linux)**: 1.12 and higher (Recommended 1.13/17.03 and above to use managed plugin)
 **Docker (Windows)**: 1.13/17.03
 
-## Dependencies
-
-The plugin uses VMCI (Virtual Machine Communication Interface) and vSockets to contact the service on ESX. The associated Linux kernel drivers are installed via the VMware Tools and its open version, namely [open-vm-tools](https://github.com/vmware/open-vm-tools), packages. Either one of these packages must hence be installed in the guest OS. It's recommended to install the most uptodate version of either of these packages as available.
-
 ## Known Issues
 
--  Volume metadata file got deleted while removing volume from VM(placed on Esx2) which is in use by another VM(placed on Esx1) [#1191](https://github.com/vmware/docker-volume-vsphere/issues/1191). It's an ESX issue and will be available in the next vSphere release.
--  Full volume name with format like "volume@datastore" cannot be specified in the compose file for stack deployment. [#1315](https://github.com/vmware/docker-volume-vsphere/issues/1315). It is a docker compose issue and a workaround has been provided in the issue.
--  Volume creation using VFAT filesystem is not working currently. [#1327](https://github.com/vmware/docker-volume-vsphere/issues/1327)
--  Plugin fails to create volumes after installation on VM running boot2docker Linux. This is because open-vm-tools available for boot2docker doesn't install the vSockets driver and hence the plugin is unable to contact the ESX service. [#1744](https://github.com/vmware/docker-volume-vsphere/issues/1744)
--  Currently "vmdk-opsd stop" just stops (exits) the service forcefully. If there are operations in flight it could kill them in the middle of execution. This can potentially create inconsistencies in VM attachement, KV files or auth-db. [#1073](https://github.com/vmware/docker-volume-vsphere/issues/1073)
--  Admin CLI volume ls command is slow to list all the volumes if the setup has more than hundreds of volumes. [#1083](https://github.com/vmware/docker-volume-vsphere/issues/1083)
-
-## Known Differences Between Linux And Windows Plugin
-- Docker, by default, converts volume names to lower-case on Windows. Therefore, volume operations involving case-sensitive names will always be handled in lower case.
+If you are running into any issue, please check out [vDVS known issue page](http://vmware.github.io/docker-volume-vsphere/documentation/known-issues.html) related with vSphere docker volume service before raising a new issue.
 
 ## Contact us
+Please reach us out for any questions or problem you are facing with respect to vDVS. Please [click here](http://vmware.github.io/docker-volume-vsphere/documentation/contactus.html) to get contact detail.
```

Comment:
Let's just provide the link here - Contact us for file feature requests and report product issues! [http://vmware.github.io/docker-volume-vsphere/documentation/contactus.html]

Reply:
done!